### DISCUSSION TITLE
Unable to replace errors, too jovial for business

### DISCUSSION THREAD

**Discussion by LorenzoSapora** [laravel/nova-issues#5809] [Ideas & Feature Requests] [answered]
Not able to replace nova errors.

Previously we could replace 404/403/etc errors via a custom component (https://medium.com/laravel-nova/laravel-nova-custom-error-pages-bd382e7007d7), but that's no longer applicable.

The default laravel error page is great, just says "error 403", but a 403 in nova "The government won't let us show you what's behind these doors…", and while I may find that funny, my clients have raised some problems with it.

Currently I'm just editing the lang file to remove the joviality, but having some method to create our own would be excellent.

**Comment by LorenzoSapora** [comment]
@crynobone Hey, is it possible with nova as is? A little feedback would make this transition from issue to discussion less jarring. Without it feeling like you're just ignoring it, like this other discussion https://github.com/laravel/nova-issues/discussions/5612

**Comment by davidhemphill** [comment, marked as answer]
It should still be possible to register `CustomError404`, `CustomError403`, and `CustomAppError` components using a custom asset.
